.github/bot-files/comments.yml

```diff
@@ -0,0 +1,13 @@
+### You can define all your comments in this file for various purposes
+# for now you'll need to difine every comment here, in the future comments will be optional.
+# please see the example for the file below
+
+prFirstTimeContributor: Many thanks for sending out the PR, Welcome to our Community
+
+prCiTrigger: Jenkins, test this
```

Comment:
is this to re-test or trigger the test itself?

Reply:
We recently changed the way to trigger our pipeline on the `kogito-cloud-operator` so for anytime we need to trigger our pipeline we use this phrase `Jenkins, test this`. This is also used for re-trigger, as that was also just phrase to run the pipeline.